### ISSUE TITLE
Keyboard focus is not visible on the next, prev buttons

### ISSUE THREAD

**Issue by jonels-msft** [readthedocs/sphinx_rtd_theme#958]
### Problem

The border for keyboard focus does not appear on Next and Previous buttons in the theme.

### Reproducible Project

https://docs.citusdata.com/en/v9.3/installation/citus_cloud.html

#### Error Logs/Results

Here is a video demonstrating the problem: https://www.youtube.com/watch?v=bt11YIssVuk

#### Expected Results

Other page elements display a border when selected for keyboard focus, so the buttons should too.

### Environment Info

- Python Version: 3.7.7
- Sphinx Version: 1.4.9
- RTD Theme Version: 0.2.4
